src/public/modules/forms/services/form-fields.client.service.js

```diff
@@ -5,14 +5,18 @@ const get = require('lodash/get')
 const {
   types: basicTypes,
 } = require('../../../../../shared/constants/field/basic')
+const { BasicField } = require('../../../../../shared/types/field')
 const {
   types: myInfoTypes,
 } = require('../../../../../shared/constants/field/myinfo')
 
 angular.module('forms').service('FormFields', [FormFields])
 
 function FormFields() {
-  this.basicTypes = basicTypes
+  // TODO: Unhide Country field once the frontend for Country field is done
```

Comment:
I thought this was addressed already. Cannot remember why it's still here. Will have to address this.